pyproject.toml

```diff
@@ -50,6 +49,9 @@ dependencies = [
     'sqlalchemy>=2.0',  # GeoAlchemy2 requires >=1.4.
     'toolz',
     'xarray>=0.9',  # >0.9 fixes most problems with `crs` attributes being lost
+    'odc-geo>=0.4.8',
```

Comment:
I think it's nice to continue to keep the packages sorted alphabetically.

Reply:
OK, I've also pinned odc-loader to <=0.5.1 because of the changes Kirill mentioned above coming in 0.6.0